src/components/chat-view/Chat.tsx

```diff
@@ -413,6 +411,7 @@ const Chat = forwardRef<ChatRef, ChatProps>((props, ref) => {
             <Plus size={18} />
           </button>
           <ChatListDropdown
+            key={currentConversationId}
```

Comment:
What is the reason for setting the key as currentConversationId?

Reply:
removed!